skore/src/skore/item/cross_validate_item.py

```diff
@@ -0,0 +1,175 @@
+    df = (
+        pandas.DataFrame(cv_results)
+        .reset_index(names="split")
+        .melt(id_vars="split", var_name="metric", value_name="score")
+    )
+
+    input_dropdown = altair.binding_select(
+        options=df["metric"].unique().tolist(), name="Metric: "
+    )
+    selection = altair.selection_point(
+        fields=["metric"], bind=input_dropdown, value="test_score"
+    )
+
+    return (
+        altair.Chart(df, title="Cross-validation scores per split")
+        .mark_bar()
+        .encode(
+            altair.X("split:N").axis(
+                title="Split number",
+                labelAngle=0,
+            ),
+            altair.Y("score:Q").axis(
+                title="Score",
+                titleAngle=0,
+                titleAlign="left",
+                titleX=0,
+                titleY=-5,
+                labelLimit=300,
+            ),
+            tooltip=["metric:N", "split:N", "score:Q"],
+        )
+        .interactive()
+        .add_params(selection)
+        .transform_filter(selection)
+        .properties(
+            width=500,
+            height=200,
+            padding=15,
+            autosize=altair.AutoSizeParams(type="pad", contains="padding"),
+        )
+    )
+
+
+def _hash_numpy(array):
+    return hashlib.sha256(array.tobytes()).hexdigest()
+
+
+# Data used for training, passed as input to scikit-learn
+Data = Any
+# Target used for training, passed as input to scikit-learn
+Target = Any
+
+
+class CrossValidationItem(Item):
+    """
+    A class to represent the output of a cross-validation workflow.
+
+    This class encapsulates the output of scikit-learn's cross-validate function along
+    with its creation and update timestamps.
+    """
+
+    def __init__(
+        self,
+        cv_results: dict,
+        estimator_info: dict,
+        X_info: dict,
+        y_info: dict,
+        plot: Any,
+        created_at: str | None = None,
+        updated_at: str | None = None,
+    ):
+        """
+        Initialize a CrossValidationItem.
+
+        Parameters
+        ----------
+        cv_results : dict
+            The dict output of scikit-learn's cross_validate function.
+        estimator_info : dict
+            The estimator that was cross-validated.
+        X_info : dict
+            A summary of the data, input of scikit-learn's cross_validation function.
+        y_info : dict
+            A summary of the target, input of scikit-learn's cross_validation function.
+        plot : Any
+            A plot of the cross-validation results.
+        created_at : str
+            The creation timestamp in ISO format.
+        updated_at : str
+            The last update timestamp in ISO format.
+        """
+        super().__init__(created_at, updated_at)
+
+        self.cv_results = cv_results
+        self.estimator_info = estimator_info
+        self.X_info = X_info
+        self.y_info = y_info
+        self.plot = plot
```

Comment:
For my underlying comments, keep in mind that all attributes of an `Item` are persisted.
They must be environment independent and should be primitive.

Actually anything can persist, because we use `diskcache` which uses `pickle` behind it.
But in the near futur, we will use another backend not based on `pickle`.

Reply:
yes, this is clear